src/robot/libraries/Process.py

```diff
@@ -466,9 +477,13 @@ def start_process(
             env=env,
             **env_extra,
         )
-        command = conf.get_command(command, list(arguments))
-        self._log_start(command, conf)
-        process = subprocess.Popen(command, **conf.popen_config)
+        actual_command = conf.get_command(command, list(arguments))
```

Comment:
This variable could still be `command`, it's clear enough if the other is `masked_command`. Using `actual_command` is fine as well if you prefer that. The `list()` call seems to be totally unnecessary. Not sure why the original code had it.